packages/lib/src/login-authenticator.ts

```diff
@@ -205,6 +188,7 @@ export class LoginAuthenticator implements Authenticator {
 		// eslint-disable-next-line no-async-promise-executor
 		return new Promise(async (resolve, reject) => {
 			while (!this.authenticationInfo && Date.now() < startTime + maxDelay) {
+				process.stderr.write('.')
```

Comment:
Is there a functional reason this is `stderr` instead of `stdout`?

Reply:
It's to avoid messing with the real output of the command since we often output JSON or YAML, we don't want to mix messages to the user like this in with those.

This article does a better job than I would explaining it under "4. Mind the streams":

https://medium.com/@jdxcode/12-factor-cli-apps-dd3c227a0e46

Reply:
Awesome, thanks for the reference.